src/actions.rs

```diff
@@ -140,7 +156,7 @@ impl<'a> Action for Step<'a> {
                         let issues = query
                             .query(
                                 &repository,
-                                name == "proposed_fcp",
+                                name.contains("fcp"),
```

Comment:
I would probably revert this change. It will run an additional HTTP query also for FCPs that don't need the additional details. In fact we're interested to show these details only for FCPs with label `proposed-final-comment-period`.